### PR TITLE
feat: check tx status

### PR DESCRIPTION
**Pull request by agraebe**
irt #608

### CODE REVIEW

File: develop/connect/use-with-clarity.md [hunk 1]
```diff
@@ -246,4 +246,18 @@ const MyComponent = () => {
...
+```
+
+## Checking transaction status
+You may want to check the status of a transaction after signing and broadcasting it. For instance, your app might want to enable certain components only _after_ a transaction was successfully processed on the blockchain.
+
+Following the steps above, you'll receive a transaction ID (`txId`) as part of the `finished` callback method once your user signed the transaction. Equipped with the ID, you can use the [Stacks Blockstack API](https://docs.blockstack.org/common/core_ref.html#stacks-blockchain-api) to request the status of the transaction.
+
+```bash
+curl 'https://sidecar.staging.blockstack.xyz/sidecar/v1/tx/<txId>'
+```
```
Comment (hstove) [inline develop/connect/use-with-clarity.md:258]:
```suggestion
```js
const response = await fetch(`https://sidecar.staging.blockstack.xyz/sidecar/v1/tx/${txId}`)
const txData = await response.json();
console.log(txData.status);
```
```

Comment (hstove) [reply develop/connect/use-with-clarity.md:258]: We should use JS instead of curl, because this page is entirely in the context of writing web apps.

**Comment by markmhx** [comment]
@hstove how's this look to approve and merge with Alex's change?

**Comment by hstove** [comment]
My suggested change was missing the final backticks (```) so if you look at the Markdown preview, it's broken. I think its a bug in the Github "suggested change" feature.

**Comment by agraebe** [comment]
@hstove fixed, thanks! can you approve please?

**Pull request merged**